src/Doctrine/ManagerRegistry.php

```diff
@@ -29,7 +29,12 @@ class ManagerRegistry implements ManagerRegistryInterface
     /**
      * @var EntityManager[]
      */
-    protected $managers;
+    protected $originalManagers;
+
+    /**
+     * @var EntityManager[]
+     */
+    protected $resetManagers = [];
```

Comment:
Hello @ghola 

Makes sense, does it work resetting the same manager multiple times? I can't see a test for this, thats why i ask.

Regards @dominikzogg